frontend/src/main/java/com/google/tripmeout/frontend/storage/PersistentStorageBindingModule.java

```diff
@@ -0,0 +1,19 @@
+package com.google.tripmeout.frontend.storage;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+
+public class PersistentStorageBindingModule extends AbstractModule {
+  @Override
+  public void configure() {
+    bind(PlaceVisitStorage.class).to(PersistentPlaceVisitStorage.class).in(Singleton.class);
+  }
+  @Provides
+  DatastoreService provideDatastoreService() {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Nit: `return DatastoreServiceFactory.getDatastoreService()` since you aren't actually using it for anything else.